Polyhedron/demo/Polyhedron/Plugins/PCA/Affine_transform_plugin.cpp

```diff
@@ -113,15 +121,16 @@ class Scene_transform_point_set_item : public Scene_item
 private:
   const Scene_points_with_normal_item* base;
   qglviewer::Vec center_;
-  CGAL::Three::Scene_item::ManipulatedFrame* frame;
+  mutable CGAL::Three::Scene_item::ManipulatedFrame* frame;
```

Comment:
More precisely, which `const` function requires a non-const access to that pointer?